src/components/Modals/Send/Form.tsx

```diff
@@ -195,15 +187,20 @@ export const Form: React.FC<SendFormProps> = ({ initialAssetId, input = '', acco
         onKeyDown={checkKeyDown}
       >
         <AnimatePresence mode='wait' initial={false}>
-          <Routes>
-            <Route path={`${SendRoutes.Select}/*`} element={selectAssetRouter} />
-            <Route path={SendRoutes.Address} element={address} />
-            <Route path={SendRoutes.Details} element={details} />
-            <Route path={SendRoutes.Scan} element={qrCodeScanner} />
-            <Route path={SendRoutes.Confirm} element={confirm} />
-            <Route path={SendRoutes.Status} element={status} />
-            <Route path='/' element={selectRedirect} />
-          </Routes>
+          {/* Instead of a redirect which would produce a flash of content, we simply render the select route in place of <Switch />ing here */}
```

Comment:
Uses wouter for sanity's sake, and (!assetId && !initialAssetId) as a directive on whether to render the select route, instead of a redirect causing issues.
Note how this is wrapped in a transition to not lose transitions on back